applications/launchpad/backend/src/grpc/mod.rs

```diff
@@ -27,16 +27,13 @@ pub struct WalletTransaction {
 }
 
 #[derive(Debug, Clone, Serialize)]
-#[serde(rename_all = "camelCase")]
```

Comment:
frontend depends on this being camelCase and will break if we remove this, why is it important that it isn't remains in snake_case ?

Reply:
@tarnas Should I apply `#[serde(rename_all = "camelCase")]` to all structs: WalletIdentity,  WalletTransaction, WalletBalance, and ImageInfo and etc.?